<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectLevel</class>
 <widget class="QWidget" name="SelectLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="__wrapper" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>90</number>
    </property>
    <property name="leftMargin">
     <number>60</number>
    </property>
    <property name="topMargin">
     <number>65</number>
    </property>
    <property name="rightMargin">
     <number>60</number>
    </property>
    <property name="bottomMargin">
     <number>130</number>
    </property>
    <item>
     <widget class="QLabel" name="__title_text">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1800</width>
        <height>95</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Marmelad</family>
        <pointsize>60</pointsize>
       </font>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="text">
       <string>Выберите уровень</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="__content_box" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1800</width>
        <height>700</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>34</number>
       </property>
       <property name="topMargin">
        <number>23</number>
       </property>
       <property name="rightMargin">
        <number>35</number>
       </property>
       <property name="bottomMargin">
        <number>47</number>
       </property>
       <item>
        <widget class="QWidget" name="__levels_box" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1731</width>
           <height>520</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>33</number>
          </property>
          <property name="leftMargin">
           <number>68</number>
          </property>
          <property name="topMargin">
           <number>39</number>
          </property>
          <property name="rightMargin">
           <number>68</number>
          </property>
          <property name="bottomMargin">
           <number>30</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="__tools_w" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>62</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QSlider" name="__level_slider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>1124</width>
              <height>60</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="__button_back">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>90</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Marmelad</family>
              <pointsize>40</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Вернуться</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
